packages/react-magma-dom/src/components/Pagination/SimplePagination.tsx

```diff
@@ -0,0 +1,190 @@
+import * as React from 'react';
+import { ButtonColor, ButtonShape, ButtonVariant } from '../Button';
+import { I18nContext } from '../../i18n';
+import { Tooltip } from '../Tooltip';
+import { ArrowBackIcon, ArrowForwardIcon } from 'react-magma-icons';
+import styled from '../../theme/styled';
+import { ThemeContext } from '../../theme/ThemeContext';
+import { NativeSelect } from '../NativeSelect';
+import { NavButton, PaginationProps } from './';
+import { useGenerateId } from '../../utils';
+import { Spacer } from '../Spacer';
+import { VisuallyHidden } from '../VisuallyHidden';
+import { transparentize } from 'polished';
+
+function buildLabelColor(props) {
+  if (props.isInverse) {
+    if (props.disabled) {
+      return transparentize(0.8, props.theme.colors.neutral100);
+    }
+    return props.theme.colors.neutral100;
+  }
+  if (props.disabled) {
+    return props.theme.colors.neutral500;
+  }
+  return props.theme.colors.neutral700;
+}
+
+const StyledWrapper = styled.div<{
+  disabled?: boolean;
+  isInverse?: boolean;
+}>`
+  display: flex;
+  align-items: center;
+  max-width: 360px;
+  select {
+    min-width: 87px;
+  }
+  label {
+    color: ${buildLabelColor};
+    font-size: ${props => props.theme.typeScale.size02.fontSize};
+    font-weight: 500;
```

Comment:
I recently went through and added font-family to every component so for consistency, do you mind adding it here too?